### ISSUE TITLE
Automatic entering after the latest update

### ISSUE THREAD

**Issue by dewaang-engminds** [microsoft/vscode#303202]
Type: <b>Bug</b>

Just now VS Code terminal was working well but as soon as i clicked restart for update automatically entering is being pressed inside my vs code terminal i am unable to do anything please look into the matter.

VS Code version: Code 1.112.0 (07ff9d6178ede9a1bd12ad3399074d726ebe6e43, 2026-03-17T18:09:23Z)
OS version: Windows_NT x64 10.0.26200
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i5-1335U (12 x 2496)|
|GPU Status|2d_canvas: enabled<br>GPU0: VENDOR= 0x8086, DEVICE=0xa7a1 [Intel(R) Iris(R) Xe Graphics], DRIVER_VENDOR=Intel, DRIVER_VERSION=32.0.101.6733 *ACTIVE*<br>GPU1: VENDOR= 0x1414, DEVICE=0x008c [Microsoft Basic Render Driver], DRIVER_VERSION=10.0.26100.7309<br>Machine model name: <br>Machine model version: <br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|23.69GB (6.19GB free)|
|Process Argv|--crash-reporter-id 8514d96d-8d75-482f-8607-db390901e097|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (46)</summary>

Extension|Author (truncated)|Version
---|---|---
aws-toolkit-vscode|ama|3.99.0
vscode-tailwindcss|bra|0.14.29
vscode-markdownlint|Dav|0.61.1
vscode-html-css|ecm|2.0.14
copilot-chat|Git|0.40.1
drawfolderstructure|jmk|1.4.6
rainbow-csv|mec|3.24.1
microsoft-testing|ms-|0.2.0
azure-dev|ms-|0.10.0
vscode-azure-github-copilot|ms-|1.0.176
vscode-azure-mcp-server|ms-|1.0.4
vscode-azureappservice|ms-|0.26.4
vscode-azurecontainerapps|ms-|0.11.0
vscode-azurefunctions|ms-|1.20.3
vscode-azureresourcegroups|ms-|0.12.3
vscode-azurestaticwebapps|ms-|0.13.2
vscode-azurestorage|ms-|0.17.1
vscode-azurevirtualmachines|ms-|0.6.10
vscode-containers|ms-|2.4.1
vscode-cosmosdb|ms-|0.32.1
vscode-edge-devtools|ms-|2.1.10
vscode-kubernetes-tools|ms-|1.3.29
debugpy|ms-|2025.18.0
python|ms-|2026.4.0
vscode-pylance|ms-|2026.1.1
vscode-python-envs|ms-|1.22.0
jupyter|ms-|2025.9.1
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.3.0
vscode-jupyter-cell-tags|ms-|0.1.9
vscode-jupyter-slideshow|ms-|0.1.6
remote-containers|ms-|0.447.0
cmake-tools|ms-|1.22.28
cpp-devtools|ms-|0.4.6
cpptools|ms-|1.30.5
cpptools-extension-pack|ms-|1.5.1
live-server|ms-|0.4.17
powershell|ms-|2025.4.0
vscode-node-azure-pack|ms-|1.8.0
windows-ai-studio|ms-|0.32.1
sqltools|mtx|0.28.5
xlsx-viewer|muh|1.7.9
vscode-yaml|red|1.21.0
docxreader|Sha|1.4.2
vscode-ai-foundry|Tea|0.18.0
pdf|tom|1.2.2

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vswsl492cf:30256860
binariesv615:30325510
nativeloc1:31344060
dwcopilot:31170013
dwoutputs:31242946
copilot_t_ci:31333650
e5gg6876:31282496
pythonrdcb7:31342333
6518g693:31463988
aj953862:31281341
6abeh943:31336334
envsactivate1:31464700
cloudbuttont:31379625
aihoversummaries_t:31469308
3efgi100_wstrepl:31403338
55ia6109:31457054
use-responses-api:31390855
je187915:31454425
ec5jj548:31422691
cp_cls_t_966_ss:31454198
a9239246:31458807
4je02754:31466945
c3h7c220:31478652
ge8j1254_inline_auto_hint_haiku:31427726
38bie571_auto:31426784
cp_cls_c_1081:31454833
ia-use-proxy-models-svc:31452481
a43f0574:31442827
e9c30283:31461165
test_treatment2:31471001
nes-conv-2-2:31477812
control_6dc23131:31467998
idci7584:31464702
edit_mode_hidden:31461530
864ei723_large_tool_results_to_disk:31460878
notips:31471632
534a6447_copy:31478748
hg17d649:31458077
chat:31457767
8hig5102:31473806
c-off:31479383
cpptoolson-v2:31475363
noaffordance1:31474146
c3257622:31473910
commenticon:31473926
ddid_t:31478206

```

</details>

**Comment by angelovdev** [comment]
[redacted]

**Comment by anthonykim1** [comment]
@angelovdev Thanks for the issue. Do you have screen reader enabled?

**Comment by dewaang-engminds** [comment]
Yes, correct, the same thing is happening to my VS Code, as presented by @angelovdev. I have restarted my laptop, reinstalled VS Code, but it's not working
Please @meganrogge, tell me the approximate time needed to fix this issue.
